src/services/message-manager/message-manager.service.ts

```diff
@@ -23,24 +23,23 @@ export class MessageManagerService {
   private _manageMessage(message: Message): void {
     if (message.guild) {
       if (message.author.bot) return
-      
-      for (const prefix of Object.values(BotPrefixesEnum)) {
-        if (message.content.startsWith(prefix)) {
-          message.channel.startTyping()
 
-          const args = message.content.slice(prefix.length).split(/ +/)
-          const command = args.shift()
+      const prefix: string = PrefixManagerService.getInstance().setPrefix(message.content)
```

Comment:
I think that this is strange to have a method called "setX" returning a value.

Reply:
Ok, I will rename it 'getPrefix' :smile: